### DISCUSSION TITLE
IDs, badges and Custom cards

### DISCUSSION THREAD

**Discussion by Gibuso** [CubicRP/Cubic#10] [Ideas]
Was just talking with someone about the PD badges. I dont know if its just me, but i REALLY dont like the current state of PD badges. Im not sure how it is with IDs, but I would like to suggest something else. 
From what i know, PD gets a badge that shows a pic of them and their name. If someone steals the SAME badge and uses it, it would say the name of the person who CURRENTLY has the badge, not the real owner of the badge. I think it would be nicer if it just showed the face and the name of the person who owns the badge. If someone would steal that badge and use it, it would say the original cops name and face instead of the name of the person who just used the badge. 
**Example**
Player 1 (cop) owns the badge - uses the badge and it says: _Name1 Surname1 Picture1_
Player 2 (someone else) steals that same badge - uses it and it says _Name1 Surname1 Picture1_
**from what i understand, it is like this right now:**
Player1 (cop) owns the badge - uses the badge and it says _Name1 Surname1 Picture1_
Player 2 (someone else) steals that same badge - uses the badge and it says _Name2 Surname2 Picture2_

The look of the badge would be same for everyone as its a standart, just the picture and the name changes.

Probably shouldnt include this, but there are badges and IDs ive seen from other cities. Im not asking to make them look the same, just the same system (if possible)
the badge would appear on the side of the screen like it is currently (if im not mistaken)

Here are examples
![Badge1](https://user-images.githubusercontent.com/100157207/177420694-e4a98f2e-c57b-422b-954e-bc243a17fa0b.jpg)
![badge2](https://user-images.githubusercontent.com/100157207/177421472-8322200c-f4a4-444c-8e61-edcb7b172364.jpg)

Its probably impossible to add pictures to everyones IDs, but having an actual ID instead of just text. just has the info in it, but it looks nicer :)
The same thing would apply as on the badges. If someone steals MY ID and SOMEONE else uses it, it will still say my info, instead of the persons who is using it.

Also, dont know if this is needed but also would be nice to be able to make like business cards or custom cards for your RP reasons. Maybe you made a small business and someone asks who you are, just show them your business card. I think that would also be a nice addition to the lawyers. Maybe let them make those cards by someones request.
![badge3](https://user-images.githubusercontent.com/100157207/177423274-ea644f1f-785f-4d9b-b94b-c0230030910b.jpg)
Image from a different city

**Comment by ChivSuroo** [comment]
The badges will be a 100%, with the cards that's actually a good idea, maybe something we implement later down the line giving people the chance to establish their characters to earn themselves a card. 

Options to retrieve the item maybe from Weazel Media magazines of such etc etc.

**Comment by hanixshah** [comment]
I do agree that citizen and government worker identification cards/badges should have an option to upload a custom character picture and whenever you use it or flex it the whole thing appears.
Secondly, if someone has a stolen id/badge, so the badge/id must show the original holder information and that's right, but I believe that there is an option to use/flex it, so it's your choice which id/badge you use/flex, either your own or the stolen one or the one owned by someone else.

**Reply by ChivSuroo** [reply]
Yes, there will be a decent ID system, and it will be item-based, which means that if someone steals your ID, they may use it to attempt to impose as you. Thank you once more for your suggestions! This will pop up on your screen too, (PNG Image)

**Reply by Gibuso** [reply]
Is it gonna be the same with PD badges? If someone steals it, it will still say my info on it?

**Reply by Tyler-Knowler** [reply]
I imagine it's gonna be a case of: the item has the name and pic etc saved to that particular item. So if someone were to steal the item then the info would still be linked to it so yes.

obviously this is just my assumption.

**Reply by Gibuso** [reply]
i really hope it is gonna be like that
